osu.Game.Rulesets.Osu/Difficulty/Skills/Flashlight.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Game.Rulesets.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Difficulty.Preprocessing;
+using osu.Game.Rulesets.Osu.Objects;
+
+namespace osu.Game.Rulesets.Osu.Difficulty.Skills
+{
+    /// <summary>
+    /// Represents the skill required to memorise and hit every object in a map with the Flashlight mod enabled.
+    /// </summary>
+    public class Flashlight : OsuStrainSkill
+    {
+        public Flashlight(Mod[] mods)
+            : base(mods)
+        {
+        }
+
+        protected override double SkillMultiplier => 0.15;
+        protected override double StrainDecayBase => 0.15;
+        protected override double DecayWeight => 1.0;
```

Comment:
You have set the decay to 1.0 which means this skill is just aggregating strain, but you still use the base implementation of `DifficultyValue` which isn't meant for this type of behaviour.
Do this to get your aggregate value as the output:
```
public override double DifficultyValue()
{
    return CurrentStrain;
}
```